blender-for-unrealengine/export/bfu_export_single_fbx_action.py

```diff
@@ -117,8 +136,9 @@ def ExportSingleFbxAction(
         my_scene_unit_settings.SetUnitForUnrealEngineExport()
         my_skeletal_export_scale = bfu_utils.SkeletalExportScale(active)
         my_skeletal_export_scale.ApplySkeletalExportScale(rrf, is_a_proxy=export_as_proxy)
-        my_action_curve_scale = bfu_utils.ActionCurveScale(rrf*active.scale.z)
-        my_action_curve_scale.ResacleForUnrealEngine()
+        if not action_curve_scale:
+            action_curve_scale = bfu_utils.ActionCurveScale(rrf*active.scale.z)
+            action_curve_scale.ResacleForUnrealEngine()
```

Comment:
When there are many Actions, the process of scaling all Actions for each Action export and then returning them to their original scale was very slow, so I made it so that it only happens once.